### PR TITLE
MentionForm feedback

### PR DESCRIPTION
**Pull request by AmeeMorris**
Little change so we have the option to display the amount of mentions on the mention form

### CODE REVIEW

**Comment by AmeeMorris** [comment]
[Make workflow notify form match comment @ing](https://trello.com/c/RprcTVwO/241-make-workflow-notify-form-match-comment-ing)

File: lib/MentionsForm/MentionsFormActions.js [hunk 1]
```diff
@@ -17,6 +17,14 @@ const MentionsFormActions = props => {
     ? ['collapse', 'link-default']
     : ['primary'];
 
+  const peopleText =
+    props.mentionCount === 1
```
Comment (altaudio) [inline lib/MentionsForm/MentionsFormActions.js]: We could have a helper function that either gives us back `person/people` based on a number input

File: lib/MentionsForm/index.js [hunk 2]
```diff
@@ -87,8 +89,12 @@ class MentionsForm extends Component {
   };
 
   updateInputValue = e => {
-    this.setState({ inputValue: e.target.value });
-    this.props.onInputChange(e.target.value);
+    const { value } = e.target;
+    const mentionCount = (value.match(pattern) || []).concat(
```
Comment (altaudio) [inline lib/MentionsForm/index.js]: I imagine there is probably a way to combine the two regexes here with an OR operator, so we only need to do the `value.match` once?

File: lib/MentionsForm/index.js [hunk 3]
```diff
@@ -87,8 +89,12 @@ class MentionsForm extends Component {
...
+      value.match(/(\B@\w+)+/gi) || []
```
Comment (altaudio) [inline lib/MentionsForm/index.js]: We could _maybe_ create a const file for regex patterns that are re-useable. I'll let you decide on if that's a good idea or not, I'm on the fence.

File: tests/MentionsForm/MentionsForm.test.js [hunk 4]
```diff
@@ -117,6 +117,14 @@ describe('Mentions Form', () => {
...
 
   test('updates the input value', () => {
+    expect(wrapper.state('mentionCount')).toEqual(0);
+    wrapper
+      .instance()
+      .updateInputValue({ target: { value: '@waffle @[plop]' } });
+    expect(wrapper.state('mentionCount')).toEqual(2);
+  });
+
+  test('updates the mentionCount', () => {
```
Comment (altaudio) [inline tests/MentionsForm/MentionsForm.test.js]: Is this the description for the above test?

File: lib/MentionsForm/MentionsFormActions.js (hunk 1 above)
Comment (AmeeMorris) [reply lib/MentionsForm/MentionsFormActions.js]: Yeah this is a tricky one, the helper exists in app, but i wasn't sure wether to add it here too. I think it makes sense to make a helpers file though

File: lib/MentionsForm/index.js (hunk 3 above)
Comment (AmeeMorris) [reply lib/MentionsForm/index.js]: There's a consts file, so I can just plop it in there for now

**Pull request merged**